src/modules/UI/Settings/selectors.js

```diff
@@ -37,13 +42,20 @@ export const getDenominations = (state: State, currencyCode: string) => {
 
 export const getDisplayDenominationKey = (state: State, currencyCode: string) => {
   const settings = getSettings(state)
-  const currencySettings = settings[currencyCode]
-  const selectedDenominationKey = currencySettings.denomination
+  let currencySettings, selectedDenominationKey
+  if (settings[currencyCode]) {
+    currencySettings = settings[currencyCode]
+    selectedDenominationKey = currencySettings.denomination
+  } else {
+    let tokenInfo = settings.customTokens.find((token) => token.currencyCode === currencyCode)
```

Comment:
why not just call this currencySettings?

Reply:
Again, hypothetically we could but I see us eventually moving away from the way the denomination data is stored in Redux's settings:
![image](https://user-images.githubusercontent.com/6249205/33865976-06139cc4-dea9-11e7-9b6b-79fc630afd57.png)